humble/distribution.yaml

```diff
@@ -4251,11 +4251,11 @@ repositories:
     release:
       packages:
       - rmw_gurumdds_cpp
-      - rmw_gurumdds_shared_cpp
+      - gurumdds_cmake_module
       tags:
         release: release/humble/{package}/{version}
       url: https://github.com/ros2-gbp/rmw_gurumdds-release.git
-      version: 3.2.0-1
+      version: 3.3.0-1
```

Comment:
@YoungJin-gurum please revert these changes back to the 1.2.0-1 release and the original package list.